### PR TITLE
[12.0][ADD] fieldservice_route_account

### PR DESCRIPTION
**Pull request by murtuzasaleh**
#382 

Depends on:
- [x] #432

### CODE REVIEW

**Comment by max3903** [comment]
@murtuzasaleh Looks good! :+1: Missing the tests.

**Comment by murtuzasaleh** [comment]
> @murtuzasaleh Looks good!  Missing the tests.

Done

**Review by max3903 — APPROVED** [review]

**Review by nelsonramirezs — APPROVED** [review]

**Comment by max3903** [comment]
@murtuzasaleh Please rebase. #432 has been merged.

File: fieldservice_route_account/tests/test_dayroute_payment.py [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+        self.company = self.env.ref('base.main_company')
+        self.payment_method_manual_in = self.env.ref(
+            "account.account_payment_method_manual_in")
+        # Create a Journal
+        self.bank_journal = self.env['account.journal'].create(
+            {'name': 'Bank US', 'type': 'bank', 'code': 'BNK68',
+             'currency_id': self.company.currency_id.id})
+
+    def test_dayroute_account(self):
+        so = self.env['sale.order'].create({
```
Comment (max3903) [inline fieldservice_route_account/tests/test_dayroute_payment.py]: @murtuzasaleh This module does not depend on sale so you need to create invoices directly.

Comment (murtuzasaleh) [reply fieldservice_route_account/tests/test_dayroute_payment.py]: Done

**Pull request merged**